regression/apps/include/test.h

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The license of `.h` file is not checked in CI now. Could you please add `**/*.h` to `.licenserc.yaml`, line 11 or 12?